### DISCUSSION TITLE
请问能否增加正文行间距与脚注、尾注行间距分别设置的功能？

### DISCUSSION THREAD

**Discussion by sinologie** [sjtug/SJTUThesis#602] [提问] [answered]
问题描述：
1. 课程论文要求正文行间距设置为1.5倍行距，但是如果直接修改clsw文件中默认的linespread会导致脚注和尾注的行间距过大，能否添加分开设置的功能？
2. LaTeX中的1.5倍行距与word中的1.5倍有差异，如果想设置成和word一样的1.5倍行距应当如何进行？

感谢您的付出！

**Comment by tanukihee** [comment, marked as answer]
> LaTeX 中的 1.5 倍行距与 word 中的 1.5 倍有差异，如果想设置成和 word 一样的 1.5 倍行距应当如何进行？

这个在 zhlineskip 里有说明

**Comment by sinologie** [comment]
> > LaTeX 中的 1.5 倍行距与 word 中的 1.5 倍有差异，如果想设置成和 word 一样的 1.5 倍行距应当如何进行？
> 
> 这个在 zhlineskip 里有说明

能烦请您详细讲一讲吗，我自己尝试换算但还是没有调成一样的。
